data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -16,23 +16,19 @@ class RnMossoroSpider(BaseGazetteSpider):
 
     def parse(self, response):
         for edition in response.css("div.edicoes-list div.col-md-3"):
-            url = edition.css("a::attr(href)").get()
             raw_date = edition.css("div.card-content p::text").get().strip()
-            date = dt.datetime.strptime(raw_date, "%d/%m/%Y").date()
-            raw_edition_number = edition.css("div.card-content h4::text").get().strip()
-            edition_number = re.findall(r"DOM N. (\d+)", raw_edition_number)
+            gazette_date = dt.datetime.strptime(raw_date, "%d/%m/%Y").date()
 
-            if date > self.end_date:
+            if gazette_date > self.end_date:
                 continue
-            elif date < self.start_date:
+            if self.start_date > gazette_date:
                 return
 
-            yield Gazette(
-                date=date,
-                edition_number=edition_number,
-                file_urls=[f"https://www.dom.mossoro.rn.gov.br{url}"],
-                is_extra_edition=False,
-                power="executive_legislative",
+            intermediary_page = edition.css("a::attr(href)").get()
+            yield scrapy.Request(
+                f"https://www.dom.mossoro.rn.gov.br{intermediary_page}",
```

Comment:
A gente está usando bastante a base da url e completando com o path relativo na mão. Seria bom evitar essa repetição.

Uma dica seria usar `response.urljoin()` para não precisar deixar o domínio hard-coded
```suggestion
                response.urljoin(intermediary_page),
```
Também vale pras linhas 39 e 54